substrait_consumer/conftest.py

```diff
@@ -41,6 +41,24 @@ def pytest_addoption(parser):
         help="A comma separated list of producers to run against.",
         choices=[x.__name__ for x in PRODUCERS]
     )
+    parser.addoption(
+        "--adhoc_producer",
+        action="store",
+        default=",".join([x.__name__ for x in []]),
```

Comment:
Is this default a fancy way of saying "empty list" or am I missing something?

Reply:
Oops, this can actually just be an empty string! I'll update it.